ext/js/background/backend.js

```diff
@@ -2152,6 +2152,21 @@ class Backend {
         return textReplacements;
     }
 
+    async _openWelcomeGuidePageOnce() {
+        if (isObject(chrome.storage) && isObject(chrome.storage.session)) {
+            chrome.storage.session.get(['openedWelcomePage']).then((result) => {
+                console.log(new Date(), 'openedWelcomePage:', result.openedWelcomePage);
```

Comment:
Currently I don't believe the extension ever does a `console.log` in a non-error condition. But I think it's fine to leave some sort of debugging information in the SW.

Maybe we could consider `console.info` for intentionally leaving informational messages (like https://stackoverflow.com/a/45766458 suggests)?

Reply:
Oh this was entirely my bad, I accidentally left this in the PR. It's probably best to remove this for now, to conform with the rest of the extension.